tests/integration/create-switch-account.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { BrowserDriver, wait, createTestSelector, setupBrowser } from './utils';
+import { WalletPage } from './page-objects/wallet.page';
+import { ScreenPaths } from '@common/types';
+import { SettingsSelectors } from './settings.selectors';
+
+jest.setTimeout(60_000);
+jest.retryTimes(process.env.CI ? 2 : 0);
+describe.only(`Create and switch account`, () => {
+  let browser: BrowserDriver;
+  let wallet: WalletPage;
+
+  beforeEach(async () => {
+    browser = await setupBrowser();
+    wallet = await WalletPage.init(browser, ScreenPaths.INSTALLED);
+  }, 10000);
+
+  afterEach(async () => {
+    try {
+      await browser.context.close();
+    } catch (error) {}
+  });
+
+  it('should be able to create a new account then switch', async () => {
+    await wallet.signUp();
+    await wallet.clickSettingsButton();
+    await wallet.page.click(createTestSelector('settings-create-an-account'));
+    await wallet.page.click(createTestSelector('create-account-done-button'));
+    const displayName = await wallet.page.textContent(
+      createTestSelector('home-current-display-name')
+    );
+    expect(displayName).toEqual('Account 2');
+    await wallet.clickSettingsButton();
+    await wallet.page.click(createTestSelector(SettingsSelectors.SwitchAccount));
+    await wallet.page.click(createTestSelector('account-0'));
+    await wait(500);
+    const displayName2 = await wallet.page.textContent(
+      createTestSelector('home-current-display-name')
+    );
+    expect(displayName2).toEqual('Account 1');
+  });
+
+  const COUNT = 3;
+  it(`should be able to create ${COUNT} new accounts then switch between them`, async () => {
+    await wallet.signUp();
+    for (let i = 0; i < COUNT - 1; i++) {
+      await wallet.clickSettingsButton();
+      await wallet.page.click(createTestSelector('settings-create-an-account'));
+      await wallet.page.click(createTestSelector('create-account-done-button'));
+    }
+
+    await wait(500);
```

Comment:
Check the docs here on actionability https://playwright.dev/docs/actionability

Some actions like click etc have a built in timer for which it'll wait for the selector to appear. Not all do, but you can also use `waitForSelector`. 

Could you use these methods instead of a hard coded wait?

Reply:
Sure I will check